src/rdb.c

```diff
@@ -3531,16 +3532,35 @@ int rdbSaveToReplicasSockets(int req, rdbSaveInfo *rsi) {
 
     /* Collect the connections of the replicas we want to transfer
      * the RDB to, which are i WAIT_BGSAVE_START state. */
-    server.rdb_pipe_conns = zmalloc(sizeof(connection *) * listLength(server.replicas));
-    server.rdb_pipe_numconns = 0;
-    server.rdb_pipe_numconns_writing = 0;
+    int connsnum = 0;
+    connection **conns = zmalloc(sizeof(connection *) * listLength(server.replicas));
+    server.rdb_pipe_conns = NULL;
+    if (!direct) {
+        server.rdb_pipe_conns = conns;
+        server.rdb_pipe_numconns = 0;
+        server.rdb_pipe_numconns_writing = 0;
+    }
+    /* Filter replica connections pending full sync (ie. in WAIT_BGSAVE_START state). */
     listRewind(server.replicas, &li);
-    while ((ln = listNext(&li))) {
+    while((ln = listNext(&li))) {
```

Comment:
This looks like a bad clang-format?
```suggestion
    while ((ln = listNext(&li))) {
```

Reply:
I haven't had the opportunity to apply the clang format yet. I suspect there are numerous other instances similar to this one. Is there a README or guide available that explains how to properly format all the new lines we've added?